pyquil/quantum_processor/qcs.py

```diff
@@ -60,9 +60,10 @@ def __repr__(self) -> str:
 
 def get_qcs_quantum_processor(
     quantum_processor_id: str,
-    client_configuration: QCSClientConfiguration,
+    client_configuration: Optional[QCSClientConfiguration],
     timeout: float = 10.0,
 ) -> QCSQuantumProcessor:
+    client_configuration = client_configuration or QCSClientConfiguration.load()
```

Comment:
Is this function ever expected to be called externally? I left this off originally because it's handled by `get_qc()`.

Reply:
oops, nvm. I was thinking of `_quantum_computer.py`